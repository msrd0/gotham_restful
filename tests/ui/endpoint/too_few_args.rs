#[macro_use]
extern crate gotham_restful;

#[derive(Resource)]
#[resource(read)]
struct FooResource;

#[read]
fn read() {}

fn main() {}
