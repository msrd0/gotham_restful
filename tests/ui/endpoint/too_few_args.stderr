error: Too few arguments
 --> tests/ui/endpoint/too_few_args.rs:9:4
  |
9 | fn read() {}
  |    ^^^^

error[E0412]: cannot find type `read___gotham_restful_endpoint` in this scope
 --> tests/ui/endpoint/too_few_args.rs:5:12
  |
5 | #[resource(read)]
  |            ^^^^ not found in this scope
